Jenkinsfile

```diff
@@ -1,127 +1,174 @@
-                        if [ ! -z \$RETCODE ]; then
-                            count_iter=75
-                            while [ \$count_iter -gt 0 ] ; do
-                                unset RETCODE
-                                echo "WARN : unable to copy package!! retrying after 5 mins"
-                                sleep 300
-                                scp ${env.buildServer}:${env.buildServerBasePath}/${env.buildServerPlatform}/${buildServerPath}/pkgs.${timeStamp}/MarkLogic-${params.ML_SERVER_BRANCH}-${timeStamp}.x86_64.rpm . || RETCODE=\$?
-                                if [ -z \$RETCODE ] ; then
-                                    echo "INFO" "Successfully copied package"
-                                    break
-                                fi
-                                let count_iter--
-                            done
-                            if [ ! -z \$RETCODE ] ; then
-                                echo "ERROR : unable to copy package"
-                                false
-                            else    
-                                echo "INFO" "Successfully copied package"
-                            fi
-                        fi
-                    else
-                        cp $ML_RPM .
+    try {
+            return match[0]
+    } catch (Exception e) {
+            return ''
+    }
+}
+
+def PRDraftCheck() {
+    withCredentials([usernameColonPassword(credentialsId: gitCredID, variable: 'Credentials')]) {
+        PrObj = sh(returnStdout: true, script:'''
+                     curl -u $Credentials  -X GET  ''' + githubAPIUrl + '''/pulls/$CHANGE_ID
+                     ''')
+    }
+    def jsonObj = new JsonSlurperClassic().parseText(PrObj.toString().trim())
+    return jsonObj.draft
+}
+
+def getReviewState() {
+    def reviewResponse
+    def commitHash
+    withCredentials([usernameColonPassword(credentialsId: gitCredID, variable: 'Credentials')]) {
+        reviewResponse = sh(returnStdout: true, script:'''
+                            curl -u $Credentials  -X GET  ''' + githubAPIUrl + '''/pulls/$CHANGE_ID/reviews
+                             ''')
+         commitHash = sh(returnStdout: true, script:'''
+                         curl -u $Credentials  -X GET  ''' + githubAPIUrl + '''/pulls/$CHANGE_ID
+                         ''')
+    }
+    def jsonObj = new JsonSlurperClassic().parseText(commitHash.toString().trim())
+    def commit_id = jsonObj.head.sha
+    println(commit_id)
+    def reviewState = getReviewStateOfPR reviewResponse, 2, commit_id
+    echo reviewState
+    return reviewState
+}
+
+def getServerPath(branchName) {
+    switch (branchName) {
+            case '10.1':
+            return 'rh7v-10-tst-bld-1.eng.marklogic.com/b10_1'
+    case '11.0':
+            return 'rh7v-i64-11-build/HEAD'
+            case '9.0':
```

Comment:
My brain just broke on this line, can we remove the extra tab/4spaces.

Reply:
I need to fix my groovy linter. For some reason it keeps inserting extra spaces.